eclass/verify-sig.eclass

```diff
@@ -426,10 +426,14 @@ verify-sig_src_unpack() {
 			verify-sig_verify_detached \
 				"${DISTDIR}/${f%.*}" "${DISTDIR}/${f}"
 		done
-	fi
 
-	# finally, unpack the distfiles
-	default_src_unpack
+		# finally, unpack the distfiles
+		if [[ ${#distfiles[@]} -gt 0 ]]; then
```

Comment:
Do we ever have a valid case for no distfiles here? I suppose the eclass currently silently permits empty `A`, though that seems like a dysfunctional case, perhaps one that should actually cause an error.

Reply:
> I suppose the eclass currently silently permits empty A

Yes, this is why I added the check for a non-empty distfiles array. I wanted to maintain current behavior.

Reply:
I can't think of a valid case of the top of my head, but I also don't think verify-sig.eclass should be responsible for catching random mistakes like an empty SRC_URI.